src/lib/forms/rjsf/RJSForm.js

```diff
@@ -56,6 +56,25 @@ export class RJSForm extends Component {
     };
   }
 
+  componentDidMount() {
+    const inputs = document.getElementsByTagName('input');
+
+    this.ignoreEnterEvent(inputs);
+  }
+
+  ignoreEnterEvent = (inputElements) => {
```

Comment:
wasn't this part of one of the previous PRs?

Reply:
it seems i may have accidentally based this branch on that PR yes :sweat_smile: